### DISCUSSION TITLE
 .

### DISCUSSION THREAD

**Discussion by RowlingJoe** [WeNeedHome/SummaryOfLoanSuspension#28] [General]
-

**Comment by zhaokai5520** [comment]
> 点击链接加入群聊【烂尾停贷互助】：https://jq.qq.com/?_wv=1027&k=o0xDYUJC
> 
> 点击链接加入频道【烂尾停贷互助】: https://qun.qq.com/qqweb/qunpro/share?_wv=3&_wwv=128&appChannel=share&inviteCode=1W70DJD&from=246610&biz=ka#/pc

群号是多少？
